Gemfile.template

```diff
@@ -11,6 +11,7 @@ gem "paquet", "~> 0.2"
 gem "pleaserun", "~>0.0.28"
 gem "rake", "~> 12"
 gem "ruby-progressbar", "~> 1"
+gem "logstash-output-elasticsearch"
```

Comment:
Is there a reason why this is special?

Reply:
elasticsearch is now needed in core for license checking, so it's no longer "just another plugin"